Makefile

```diff
@@ -0,0 +1,78 @@
+# =================================================================
+#
+# Work of the U.S. Department of Defense, Defense Digital Service.
+# Released as open source under the MIT License.  See LICENSE file.
+#
+# =================================================================
+
+.PHONY: help
+help:  ## Print the help documentation
+	@grep -E '^[\/a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+#
+# Terraform
+#
+
+.PHONY: format_terraform
+format_terraform: ## Format terraform files
+	bash scripts/format-terraform
+
+.PHONY: update_docs
+update_docs: ## Update docs
+	bash scripts/update-docs
+
+#
+# Go building, formatting, testing, and installing
+#
+
+fmt:  ## Format Go source code
+	go fmt $$(go list ./... )
+
+.PHONY: imports
+imports: bin/goimports ## Update imports in Go source code
+	bin/goimports -w -local github.com/gruntwork-io/terratest,github.com/aws/aws-sdk-go,github.com/dod-iac,github.com/dod-iac/terraform-aws-sqs-queue $$(find . -iname '*.go')
```

Comment:
This file needs to be updated to change the `terraform-aws-sqs-queue` to your repo.

Reply:
```suggestion
	bin/goimports -w -local github.com/dod-iac,github.com/dod-iac/terraform-aws-lambda-function $$(find . -iname '*.go')
```

Reply:
After you've fixed this run `make imports` to fix up the imports on your go file.